packages/discord.js/typings/index.d.ts

```diff
@@ -229,6 +229,8 @@ declare module 'node:events' {
   }
 }
 
+type JSONValue = string | number | object | boolean | JSONValue[];
```

Comment:
Shouldn't this be exported?

Reply:
Sure, I just assumed it was for internal use only.

Reply:
Well, it's documented in the docs without `@private`, but people may find it useful to have for handling the field's type when handling `notFound`.

Reply:
Just exported the type and sorted it into the typedef region in the types!